proposals/nixpkgs-security.md

```diff
@@ -0,0 +1,278 @@
+
+## Why is this technology critical? Please explain to us the relevance of this technology.
+
+This year Nix has seen its 20th anniversary. We estimate the community to consist of 100 mostly volunteer developers, 2 000 contributors, more than 15 000 active users, and 100 companies using and relying on Nix. [Recent social media activity](https://www.youtube.com/watch?v=fuWPuJZ9NcU) reaches on the order of 100 000 people. The Nix ecosystem is on an [exponential growth trajectory](https://ossinsight.io/analyze/NixOS/nixpkgs#overview) in terms of user adoption and development activity at the periphery.
+
+Nix has the potential to become a default in the software development tool stack, by drastically reducing the time required to start working on software development projects – written in any language – and the cost of maintaining and sharing build setups and development environments. These can build on top of Nixpkgs, which today supplies the largest collection of free software in existence ready for use. Packages and services can be configured and deployed through NixOS more flexibly, reliably, efficiently than with any comparable tool. The Nix ecosystem can enable almost indefinite reproducibility of builds and configurations: [Nix and legacy enterprise software development: an unlikely match made in heaven](https://talks.nixcon.org/nixcon-2022/talk/QQPBFW/).
+
+It serves as the platform accelerating and keeping usable countless software experiments that would otherwise be intractable or have long become inaccessible as their environments grow past them.
+
+Moreover, the public [Nix binary cache](http://cache.nixos.org/) provides operational software artefacts from more than a decade back, some of which cannot be obtained otherwise any more.
+
+In short, Nix reduces the global cost of software production and long-term use by a tangible amount.
+
+Now this is said, bringing the security posture of nixpkgs to another level is crucial to improve the whole ecosystem and provide defense in depths to measures even if we cannot control all the package's contents.
+
+Here, we focus on reducing our reliance on foreign binaries to recompile nixpkgs from scratch and improve our trust following [Reflections on Trusting the Trust](https://www.cs.cmu.edu/~rdriley/487/papers/Thompson_1984_ReflectionsonTrustingTrust.pdf), ensuring we are able to trust what we boot exploiting the existing security platform and finally trusting that we always strive to use the most up-to-date, patched software whenever it is available in a sustainable fashion.
```

Comment:
Is that what this proposal is about? The bullets later on don't have any points regarding bootstrapping AFAICT.

Reply:
I need @emilytrau to fill the blanks in some area.

Reply:
Some more added in this pad https://pad.lassul.us/gwTw1dwRRZeMc5Bmd7-EXw?edit